src/main/engine/SceneObject/src/TextObject.cpp

```diff
@@ -148,7 +151,6 @@ void TextObject::setMessage(string message) {
     // Perform cleanup on existing VAOs
     gfxController_->bindVao(0);
     for (auto vao : vaos_) {
-        gfxController_->disableVertexAttArray(0);
```

Comment:
This caused issues on macOS so removing. Not really necessary for proper cleanup.